aws-postgresql.yml

```diff
@@ -15,45 +15,46 @@
 version: 1
 name: csb-aws-postgresql
 id: fa6334bc-5314-4b63-8a74-c0e4b638c950
-description: Amazon RDS for PostgreSQL
-display_name: Amazon RDS for PostgreSQL
+description: Beta - Amazon RDS for PostgreSQL
+display_name: Amazon RDS for PostgreSQL (Beta)
 image_url: https://d1.awsstatic.com/rdsImages/postgresql_logo.6de4615badd99412268bc6aa8fc958a0f403dd41.png
 documentation_url: https://aws.amazon.com/rds/postgresql/
 support_url: https://aws.amazon.com/rds/postgresql/
-tags: [aws, postgresql, postgres, preview]
+tags: [aws, postgresql, postgres, beta]
 plan_updateable: true
 plans:
 - name: small
   id: ffc51616-228b-41bd-bed1-d601c18d58f5
-  description: 'PostgreSQL 11, minimum 2 cores, minimum 4GB ram, 5GB storage'
-  display_name: "small"
+  description: 'Beta - PostgreSQL 11, minimum 2 cores, minimum 4GB ram, 5GB storage'
+  display_name: "small (Beta)"
   properties:
     cores: 2
     postgres_version: 11
     storage_gb: 5
     subsume: false
 - name: medium
   id: e64d07f9-ceb2-40a6-abd9-391047fa3cf5
-  description: 'PostgreSQL 11, minimum 4 cores, minimum 8GB ram, 10GB storage'
-  display_name: "medium"
+  description: 'Beta - PostgreSQL 11, minimum 4 cores, minimum 8GB ram, 10GB storage'
+  display_name: "medium (Beta)"
   properties:
     cores: 4
     postgres_version: 11
     storage_gb: 10
     subsume: false
 - name: large
   id: 48baef10-a14c-4ae1-aab5-25f26eba941a
-  description: 'PostgreSQL 11, minimum 8 cores, minimum 16GB ram, 20GB storage'
-  display_name: "large"
+  description: 'Beta - PostgreSQL 11, minimum 8 cores, minimum 16GB ram, 20GB storage'
+  display_name: "large (Beta)"
   properties:
     cores: 8
     postgres_version: 11
     storage_gb: 20
     subsume: false
 - name: subsume
   id: 886cb524-1d4f-11eb-b35f-8ba96a9085d7
-  description: 'Subsume control of an existing Postgre Database'
-  display_name: "Subsume"
+  description: 'Beta - Subsume
+  control of an existing Postgre Database'
```

Comment:
thoughts: probably there is an extra "\n" character

Reply:
done, thanks!